python/GangaLHCb/Lib/Splitters/GaussSplitter.py

```diff
@@ -77,7 +77,11 @@ def split(self, job):
             first = self.firstEventNumber + i * self.eventsPerJob + 1
             opts = 'from Gaudi.Configuration import * \n'
```

Comment:
Is this line still needed? The `GenInit` is used but I don't see anything other than the `ApplicationMgr` which may be imported here, or am I mis-reading the code?